Cargo.toml

```diff
@@ -79,12 +77,9 @@ tracing = { workspace = true }
 url = { workspace = true, features = ["serde"] }
 uuid = { workspace = true, features = ["v4", "v5"] }
 
-# Transitive dependency
-ahash = { workspace = true }
-time = { workspace = true }
-
 [dev-dependencies]
 astarte-device-sdk-derive = { workspace = true }
+async-trait = { workspace = true }
```

Comment:
Is it still necessary for dev-dependencies?

Reply:
Yes because I implemented the MessageHubServer trait to test the grpc connection, and that one is still using the async_trait crate.